src/main/java/uk/gov/hmcts/reform/notificationservice/model/out/NotificationInfo.java

```diff
@@ -7,6 +7,10 @@
 import java.time.Instant;
 
 public class NotificationInfo {
+
+    @JsonProperty("id")
+    public final String id;
```

Comment:
why not just long?

Reply:
I thought we like string in json response